src/utils/documentUtils.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import JSZip from "jszip";
+import { docPropsCoreXmlPath, docPropsRootElement } from "../constants";
+
+export const createOrUpdateProperty = (
+    doc: Document,
+    parent: Element,
+    property: string,
+    value: string | null | undefined
```

Comment:
small pattern mix - once we do `x?: string | null` and another time `x: string | null | undefined`